.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+version: 2.1
+orbs:
+  node: circleci/node@3.0.1
+
+jobs:
+  build-lint-test:
+    parameters:
+      version:
+        type: string
+    docker:
+      - image: circleci/node:<< parameters.version >>
+    steps:
+      - checkout
+      - run: npm install
+      - run: npm run lint:js
+      - run: npm test
```

Comment:
Adding `npm run build` here would be nice to prove that build script is also working with that node version and generating the assets

Reply:
Addressed in #278